preseed_ubuntu.cfg

```diff
@@ -1,13 +1,12 @@
 d-i debian-installer/locale string en_US
 d-i debian-installer/language string en
 d-i debian-installer/country string US
-d-i debian-installer/add-kernel-opts string console=ttyS0,115200 earlyprint=serial,115200,ttyS0
```

Comment:
Jak wyżej

Reply:
Te argumenty są podawane już do instalatora, a potem są kopiowane do systemu. Jeśli tu je ustawię to są niepotrzebnie zdublowane. Ustawiłem w poprzedniej wersji bo myślałem, że z jakiegoś powodu ich nie kopiuje, ale okazało się że wtedy problemem był GRUB.